### PR TITLE
Added a LegacyEntity for test

### PR DESCRIPTION
**Pull request by BobBooth**
Added with supervision

### CODE REVIEW

**Comment by BobBooth** [comment]
@abouffard - I think these two changes wrap up both Space and Space Missile.

**Comment by BobBooth** [comment]
@abouffard - Just added the first Sea Subsurface item. Would appreciate if you gave it a look.

**Comment by BobBooth** [comment]
@abouffard - finished Sea Subsurface, ready for review.

**Comment by BobBooth** [comment]
@abouffard - finished Sea Surface

File: instance/jmsml_D_Sea_Subsurface.xml [hunk 1]
```diff
@@ -379,33 +379,57 @@
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_SUBMARINE_POSSIBLE_SUBMARINE_LOW_1_MOD_SYM" Label="S*UPS1----*****" EntityID="MILITARY" EntityTypeID="SUBMARINE" ModifierOneID="POSSIBLE_SUBMARINE_LOW_1_MOD">
       <LegacyFunctionCode Name="2525C">S1----</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_SUBMARINE_POSSIBLE_SUBMARINE_LOW_1_MOD_LE" Label="WAR.SBSUF.SUB.POSS1" Graphic ="S-U-S1---.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Sea_Subsurface.xml]: @BobBooth - spaces usually aren't used, for consistency, between the attribute name and its value.  For example Graphic ="S-U-S1----.svg" should be Graphic="S-U-S1----.svg".

File: instance/jmsml_D_Space.xml [hunk 2]
```diff
@@ -303,9 +303,15 @@
...
     <LegacySymbol ID="MILITARY_SATELLITE_SYM" Label="S*PPS-----*****" EntityID="MILITARY" EntityTypeID="SATELLITE">
       <LegacyFunctionCode Name="2525C">S-----</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_SATELLITE_LE" Label="WAR.SPC.SAT" Graphic ="S-P-S-----.svg">
+		<LegacyFunctionCode Name="2525C">S-----</LegacyFunctionCode>
+	  </LegacyEntity>		
     </LegacySymbol>
     <LegacySymbol ID="MILITARY_SPACE_STATION_SYM" Label="S*PPT-----*****" EntityID="MILITARY" EntityTypeID="SPACE_STATION">
       <LegacyFunctionCode Name="2525C">T-----</LegacyFunctionCode>
+	  <LegacyEntity ID="MILITARY_SPACE_STATION_LE" Label="WAR.SPC.SST" Graphic ="S-P-T-----.svg">
+		<LegacyFunctionCode Name="2525C">S-----</LegacyFunctionCode>
```
Comment (abouffard) [inline instance/jmsml_D_Space.xml]: @BobBooth - this should be "T-----".

File: instance/jmsml_D_Sea_Subsurface.xml (hunk 1 above)
Comment (BobBooth) [inline instance/jmsml_D_Sea_Subsurface.xml]: @abouffard - Got it.  Just cleaned those up.

File: instance/jmsml_D_Space.xml (hunk 2 above)
Comment (BobBooth) [inline instance/jmsml_D_Space.xml]: @abouffard - fixed

**Comment by BobBooth** [comment]
@abouffard - I have made those updates, I think it is ready to merge.

**Comment by abouffard** [comment]
Thanks @BobBooth

**Pull request merged**